File: codebees_custom_app/login_redirect.js
frappe.ui.login = {
    on_login: function (user) {
        frappe.call({
            method: 'your_custom_method_to_get_employee_profile_link',
            args: { user: user },
            callback: function (r) {
                if (r.message && r.message.redirect_url) {
                    window.location.href = r.message.redirect_url;
                }
            }
        });
    }
};

File: codebees_custom_app/login_redirect.py
# In your custom Python code
import frappe

@frappe.whitelist()
def your_custom_method_to_get_employee_profile_link(user):
    # Assuming you want to redirect employees to their profile page
    employee_profile_link = f'/app/employee/{user}'
    return {'redirect_url': employee_profile_link}
